tools/autograd/gen_python_functions.py

```diff
@@ -84,6 +84,7 @@
     '_cholesky.*', '_triangular_solve.*', '_qr.*', '_symeig.*', '_svd.*',
     'slice', 'randint(_out)?',
     'item', '_local_scalar_dense', 'to',
+    '_to_copy',
```

Comment:
Is that necessary?
You can use the python version to test it if needed?

Reply:
The python codegen chokes on the TensorOptions-like args so this is actually necessary

Reply:
Ho ok!